includes/admin/home/tasks/class-sensei-home-tasks.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * File containing the Sensei_Home_Tasks class.
+ *
+ * @package sensei-lms
+ * @since $$next-version$$
+ */
+
+/**
+ * Sensei_Home_Tasks class.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_Home_Tasks {
+
+	/**
+	 * The list of actual tasks.
+	 *
+	 * @var Sensei_Home_Task[]
+	 */
+	private $items;
+
+	/**
+	 * Class constructor.
+	 *
+	 * @param Sensei_Home_Task[] $items The actual tasks.
+	 */
+	public function __construct( array $items ) {
+		$this->items = $items;
+	}
+
+	/**
+	 * The the actual tasks.
```

Comment:
```suggestion
	 * The actual tasks.
```

Just a minor typo. :)